Domain Connect Spec Draft.adoc

```diff
@@ -175,13 +178,12 @@ a domain name.
 
 === Case Sensitivity
 
-All values are case sensitive, with the following exceptions:
+All JSON object names, JSON object values, variable names, query parameters and query parameter values are case sensitive, with the following exceptions:
 
-* The following data passed in query strings:
-** The value of the serviceId, providerId, domain and host
-** All variable names
-* Variable names in templates are case insensitive
-* All variable names/keys in JSON documents are case insensitive
+* When retrieving the template from the DomainConnect Template Repository, the
+providerId and serviceId are all lower case. Regardless of how the template is

```

Comment:
This sentence is confusing and only relevant to the file repository, which is described together with it's case-sensitiveness properties above. I would just remove this bullet point.